configure.ac

```diff
@@ -134,5 +124,27 @@ AC_ARG_ENABLE(gui, [AS_HELP_STRING([--enable-gui],
                  [enable_gui=false]
                  )
 
+# Check for termcap and readline or bypass checking for the libraries.
+AC_ARG_ENABLE([libless], [AS_HELP_STRING([--enable-libless],
+                            [use a libreadline replacement with less features])],
+                 [enable_libless=true
```

Comment:
This is actually wrong. The last two actions are actually `argument presented` and `argument not presented`, but not `argument enabled` and `argument disabled`.